sci-libs/vtk/Manifest

```diff
@@ -1,4 +1,12 @@
+DIST 325acbb1a74ae27148d76399c2e7c7c5 6421 BLAKE2B 87528a3bc1aeed7293ac5973488696e836ab8a5ff287990dad6aab221aac679cb3ace5aa602ae5ac46e6ad07e1bb6847dbc7619f8741a789121d60b58835e2d4 SHA512 a3ca7836c5dd9c55ee11f963e2bf192ebc4a17895b71f2d69be24163d9d36a759778446d3f05730a0f3b84df2ad8fa143be05e0e8c2b16c22f64e3944958e3d3
```

Comment:
Let's fix these filenames

Reply:
Ok, I'm trying to find out to what option they belong and name them accordingly.

Reply:
Commented the tests which need those downloads. Point here is, the files need to be saved with their hash value as name to be found by cmake (see the files in `${S}/.ExternalData` after unpacking).

Reply:
You can't rename them in src_unpack or (I guess preferred) prepare or similar?

Reply:
You mean, renaming them twice? First at download time to some name and later in src_unpack or src_prepare from some name back to the hash? Does this make sense, just to keep the Manifest clean from meaningless names?